ext/src/BatchStatement.c

```diff
@@ -94,9 +94,9 @@ PHP_METHOD(BatchStatement, add)
 
 #if PHP_MAJOR_VERSION >= 7
   ZVAL_PTR(&entry, batch_statement_entry);
-  zend_hash_next_index_insert(&self->statements, &entry);
+  zend_hash_next_index_insert(&self->data.batch.statements, &entry);
 #else
-  zend_hash_next_index_insert(&self->statements,
+  zend_hash_next_index_insert(&self->data.batch.statements,
```

Comment:
Shouldn't we be using the macros here (e.g. **`PHP5TO7_ZEND_HASH_NEXT_INDEX_INSERT`**)?